_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/templates/node/node--video-resource--full.html.twig

```diff
@@ -74,18 +74,17 @@
   </div>
 
   {% if ( content.field_video_resource_tags|render|striptags|trim is not empty ) or ( content.field_related_content|render|striptags|trim is not empty ) %}
-  <section class="video-page related-content">
-    <div class="pf-l-grid">
-      <div id ="video-related-cont" class="pf-l-grid__item pf-m-12-col" data-tags="{{ video_tags|json_encode|replace('\"', '\'') }}">
-        <h4>MORE LIKE THIS</h4>
-        {{ content.field_related_content }}
-        <div class="video-related-content video-related-content-list"></div>
-      </div>
+  <section class="video-page rhd-c-card-grid">
+    <div class="pf-l-flex" >
+      <h2 class="pf-c-title">More Like This</h4>
```

Comment:
This is the line where the properties do not match. Everything else looks great!

Reply:
I pushed a change for this.  Thank you for catching that.